### PR TITLE
fixed announcements accessibility

### LINKED ISSUE #241: Announcement component Accessibility
Currently the timestamp and body of each announcement on the announcement page are given HTML header `component` props, meaning they build as `h3` and `h4` components in the DOM. These semantically should not be headers.

image

We can also mess with the timestamp and body Typography `variant` props so we don't have the same bold text as the header default styles and we can match "body" text across the HOTH website (i.e. text that is not a header is not bolded and is of a different font family.) This can be seen below in the example. The top is the original styling and the bottom changes the variant to `subtitle` and `body` [Typography variants](https://mui.com/api/typography/)

image

### PR DESCRIPTION
**Pull request by nathanzzhang**
fixes #241

### CODE REVIEW

**Review by jakobreinwald — APPROVED** [review]
Looks good to me!

File: src/components/AnnouncementsPage/Announcement.js [hunk 1]
```diff
@@ -28,13 +28,13 @@ function Announcement({ subject, timestamp, body }) {
 
 	return (
 		<Container>
-			<Typography variant='h4' component='h2' className={classes.subject}>
+			<Typography variant='h5' component='h2' className={classes.subject}>
 				{subject}
 			</Typography>
-			<Typography variant='h6' component='h3' className={classes.timestamp}>
+			<Typography variant='subtitle1' component='subtitle' className={classes.timestamp}>
```
Comment (jodymlin) [inline src/components/AnnouncementsPage/Announcement.js:34]:
```suggestion
			<Typography variant='subtitle1' className={classes.timestamp}>
```

File: src/components/AnnouncementsPage/Announcement.js [hunk 2]
```diff
@@ -28,13 +28,13 @@ function Announcement({ subject, timestamp, body }) {
...
                 Posted on {date} at {time} PST
 			</Typography>
-			<Typography variant='h6' component='h4' className={classes.body}>
+			<Typography variant='body2' component='body' className={classes.body}>
```
Comment (jodymlin) [inline src/components/AnnouncementsPage/Announcement.js:37]:
```suggestion
			<Typography variant='body2' className={classes.body}>
```

**Review by jodymlin — CHANGES_REQUESTED** [review]
minor changes

**Review by jodymlin — APPROVED** [review]

**Pull request merged**